src/components/AssetsTable/index.jsx

```diff
@@ -108,9 +142,43 @@ export class AssetsTable extends React.Component {
     return this.trashcanRefs[focusAsset.id];
   }
 
-  getImageThumbnail(thumbnail) {
-    const baseUrl = this.props.courseDetails.base_url ? this.props.courseDetails.base_url : '';
-    return thumbnail ? (<img src={`${baseUrl}${thumbnail}`} alt="Description not available" />) : 'Preview not available';
+  getUpdatingLockButton(asset) {
```

Comment:
Nit: Could we, maybe, choose a consistent name for the concept of "loading?" Either toggling, updating, loading, requesting, fetching, whatever. I'm finding I have to kind of... translate them all to the same concept, and having a single word for the concept of "updating" would be easier, I think. I think "loading" is a good catch-all terms for various kinds of requests (e.g. can't toggle a delete), but up to you!